doc/particles.md

```diff
@@ -362,12 +358,12 @@ flareAnimation.height = flareSize;
 // Somewhere in game
 game.add(
   ParticleComponent(
-    particle: FlareParticle(flare: flareAnimation);
-  )
+    particle: FlareParticle(flare: flareAnimation),
+  ),
 );
 ```
 
-## ComputedParticle
+## Computed Particle
```

Comment:
Isn't it better to keep it without the space? Then it will pop up when you search for it

Reply:
Fixed.